packages/mds-audit-api/service.ts

```diff
@@ -87,9 +88,8 @@ export async function writeAuditEvent(event: AuditEvent): Promise<Recorded<Audit
   return result
 }
 
-export async function readDevice(device_id: UUID, provider_id: UUID): Promise<Recorded<Device>> {
-  const result: Recorded<Device> = await db.readDevice(device_id, provider_id)
```

Comment:
I don't understand what `Recorded` was doing here. One possibility that occurs to me is that it was wrapping the device in some way needed by Audit API. If so, do we still need it?

If not, why don't we need it anymore?

Reply:
Was that type because of the added id and timestamp on the database table, absent from `Device`?

Reply:
It's still there if you look at the type of `DeviceDomainModel`, which is what's now being returned. I do need to update the type definition from `Device` to `DeviceDomainModel`, though, thanks.